common/build/eslint-plugin-fluid/src/test/example/no-unchecked-record-access/fileWithOnlyRecordAccess.ts

```diff
@@ -52,11 +60,23 @@ if ("a" in indexedRecordOfStrings) {
 	indexedRecordOfStrings.a.length; // ok: Accessing length of property inside an 'in' check, 'a' is guaranteed to be defined
 }
 
+function recordAFnExpectsString(record: IndexSignatureType): string {
+	return record.a; // defect: Returning index property 'a' directly, but 'a' might not be present
+}
+
+function recordAFnExpectsStringOrUndefined(record: IndexSignatureType): string | undefined {
+	return record.a; // ok: Returning index property 'a' to string or undefined variable, 'a' might not be present
+}
+
```

Comment:
The function tests should take values and not records.
Passing a parameter is like making a variable assignment (but not exactly the same). And that is why testing is important.